.ci/test_on_docker.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash -xe
+
+# Thanks to:
+# https://github.com/opensciencegrid/htcondor-ce/tree/master/tests
+
+OS_VERSION=$1
+PY_VER=$2
+ROOT_VER=$3
+
+echo OS_VERSION $OS_VERSION
+echo PY_VER $PY_VER
+echo ROOT_VER $ROOT_VER
+
+sudo yum -y install man
+
+uname -a
+whoami
+
+export BUILD_HOME=/home/daqbuild
+export DATA_PATH=/data
```

Comment:
Was the dependency on `$ELOG_PATH` removed from the plotting macros?

Reply:
I don't think so, all these vars are set in the `docker` image, so here was just a redundant check from before